cypress/integration/onboarding.spec.js

```diff
@@ -123,7 +120,9 @@ context("onboarding", () => {
 
       // Success screen.
       cy.pick("urn").contains(radIdRegex).should("exist");
+      cy.pick("go-to-profile-button").click();
 
+      cy.log("reset session");
```

Comment:
This is cool 👍

I wonder if we should be using `.log` everywhere instead of adding comments in the specs about what's going on? 🤔

Reply:
I think we can start by replacing `console.log` in the support commands. This would be very beneficial.